packages/by-name/kata/kata-runtime/package.nix

```diff
@@ -11,19 +11,18 @@
 
 buildGoModule rec {
   pname = "kata-runtime";
-  version = "3.13.0";
+  version = "3.14.0";
 
   src = applyPatches {
     src = fetchFromGitHub {
       owner = "kata-containers";
       repo = "kata-containers";
       rev = version;
-      hash = "sha256-xBEK+Tczc4MVnETx5sV9sb5/myxLeP7YDDigTroN4Lg=";
+      hash = "sha256-OcKsD9y1kb4IgYyCsqW3aHMB4pDFlIWfAUzoUJVe8pQ=";
     };
 
     patches = [
-      ./0001-govmm-Directly-pass-the-firwmare-using-bios-with-SNP.patch
-      ./0002-emulate-CPU-model-that-most-closely-matches-the-host.patch
+      ./0001-emulate-CPU-model-that-most-closely-matches-the-host.patch
```

Comment:
Implemented upstream in https://github.com/kata-containers/kata-containers/commit/c1ca49a66c16e1a1e817591e1bba085c8fd0ffe6